content/guides/spring/spring-cloud-connectors.md

```diff
@@ -0,0 +1,243 @@
+---
+title:  "What are Spring Cloud Connectors?"
+linkTitle: "Spring Cloud Connectors"
+subsection: "Spring Cloud Connectors"
+weight: 5
+topics:
+- Spring
+tags:
+- Kafka
+- Spring
+- Spring Data
+- Spring Cloud
+- Spring Cloud Connectors
+patterns:
+- API
+---
+
+[Spring Cloud Connectors](https://cloud.spring.io/spring-cloud-connectors/) is a project that simplifies the process of connecting Spring applications to services in cloud platforms and gaining operational awareness of those platforms. It is designed for extensibility: you can use one of the provided cloud connectors or write one for your cloud platform, you can use the built-in support for commonly-used services (like relational databases, MongoDB, Redis, RabbitMQ), or extend Spring Cloud Connectors to work with your own services.
+
+## Cloud Platform Support
+
+Out of the box, Spring Cloud Connectors offers support for the Cloud Foundry and Heroku platforms. You can also extend Spring Cloud Connectors to provide support for other cloud platforms and providers.
+
+Spring Cloud Connectors uses the `CloudConnector` interface to provide cloud platform support. A `CloudConnector` implementation for a particular cloud platform is responsible for detecting when the application is running in that cloud platform, obtaining information about the application from the cloud platform, and obtaining information about the services that are bound to the application.
+
+## Cloud Service Support
+
+You can extend Spring Cloud Connectors to support additional services, including services that are specific to your own environment or application. Spring Cloud Connectors uses two interfaces to provide cloud service support:
+
+* `ServiceInfo` models the information required to connect to the service. In the case of a database service, a ServiceInfo implementation might include fields for host, port, database name, username, and password; in the case of a web service, it might include fields for URL and API key.
+
+*`ServiceInfoCreator` creates `ServiceInfo` objects based on the service information collected by a cloud connector. A ServiceInfoCreator implementation is specific to a cloud platform.
```

Comment:
Space missing between "*" and "`ServiceInfoCreator`".

It should be:
* `ServiceInfoCreator`